### PR TITLE
add boosted frame notes to usage FAQ

### PR DESCRIPTION
**Pull request by RTSandberg**
Add some notes about using the boosted frame to the usage FAQ

### CODE REVIEW

File: Docs/source/usage/faq.rst [hunk 1]
```diff
@@ -27,6 +27,17 @@ We change the default in ``cmake/dependencies/AMReX.cmake``.
 Note that the tiny profiler adds literally no overhead to the simulation runtime, thus we enable it by default.
 
 
+What do I need to know about using the boosted frame?
+-----------------------------------------------------
```
Comment (ax3l) [inline Docs/source/usage/faq.rst:39]:
```suggestion
What do I need to know about using the boosted frame?
-----------------------------------------------------

```

File: Docs/source/usage/faq.rst [hunk 2]
```diff
@@ -27,6 +27,17 @@ We change the default in ``cmake/dependencies/AMReX.cmake``.
...
+The boost transformations are calculated internally to WarpX, so the input deck can be designed in the lab frame.  Little modification is needed beyond setting ``warpx.gamma_boost``.
+There are new instabilities that arise (see :ref:`boosted frame theory <theory-boostedframe>`) that require different algorithms for stability; please see the :ref:`boosted examples <usage-examples>` for more help.
+That said, there are some practical items worth considering that we list here to assist in designing and analyzing boosted frame simulations:
+
+- The boosted frame simulation begins at boosted time :math:`t'=0`.
+- Best practice is to set the upper :math:`z` limit of the problem domain to 0 with ``geometry.prob_hi``.
+Otherwise it is possible that the boosted simulation begins with downstream objects unexpectedly in the problem domain before there is a chance to correctly calculate all interactions.
+
```
Comment (ax3l) [inline Docs/source/usage/faq.rst:52]: Let's add a link to the in-depth section on boosted frame: https://warpx.readthedocs.io/en/latest/theory/boosted_frame.html aka ref anchor `theory-boostedframe`

Comment (RTSandberg) [reply Docs/source/usage/faq.rst:52]: done

**Review by EZoni — APPROVED** [review]
Checked offline with @RTSandberg, everything that was discussed with Jean-Luc, Remi and Axel has been added, so I can merge this first FAQ notes. We can improve on this as needed. Thanks for adding this.

File: Docs/source/usage/faq.rst [hunk 3]
```diff
@@ -27,6 +27,18 @@ We change the default in ``cmake/dependencies/AMReX.cmake``.
...
 
+What do I need to know about using the boosted frame?
+-----------------------------------------------------
+
+The boost transformations are calculated internally to WarpX, so the input deck can be designed in the lab frame and no modification to the physical set-up is needed.
+There are new instabilities that arise (see :ref:`boosted frame theory <theory-boostedframe>`) that require different algorithms for stability; please see the :ref:`boosted examples <usage-examples>` for more help.
+That said, there are some practical items worth considering that we list here to assist in designing and analyzing boosted frame simulations:
+
+- The boosted frame simulation begins at boosted time :math:`t'=0`.
+- Best practice is to set the upper :math:`z` limit of the problem domain to 0 with ``geometry.prob_hi``.   Things moving to the right should start with `z < 0` and things stationary (moving to the left in the boosted frame) should start with `z > 0`.
```
Comment (EZoni) [inline Docs/source/usage/faq.rst]: Actually, only one thing, could you maybe use the `:math:` environment here too?
```suggestion
- Best practice is to set the upper :math:`z` limit of the problem domain to 0 with ``geometry.prob_hi``.   Things moving to the right should start with :math:`z < 0` and things stationary (moving to the left in the boosted frame) should start with :math:`z > 0`.
```

File: Docs/source/usage/faq.rst [hunk 4]
```diff
@@ -27,6 +27,32 @@ We change the default in ``cmake/dependencies/AMReX.cmake``.
...
+
+- Ions must be explicitly included
+- Best practice is to separate counter-propagating objects; things moving to the right should start with :math:`z <= 0` and things stationary or moving to the left (moving to the left in the boosted frame) should start with :math:`z > 0`
+- Don't forget the general design principles listed above
+- The boosted frame simulation begins at boosted time :math:`t'=0`
+- Numerics and algorithms need to be adjusted, as there are numerical instabilities that arise in the boosted frame. For example, setting ``particles.use_fdtd_nci_corr=1`` for an FDTD simulation or setting ``psatd.use_default_v_galilean=1`` for a PSATD simulation. Be careful as this is overly simplistic and these options will not work in all cases.  Please see the :ref:`input parameters documentation <running-cpp-parameters>` and the :ref:`examples <usage-examples>` for more information
+
+An in-depth discussion of the boosted frame is provided in the :ref:`moving window and optimal Lorentz boosted frame <theory-boostedframe>` section.
+
+
```
Comment (ax3l) [inline Docs/source/usage/faq.rst:53]:
```suggestion
```

**Review by ax3l — APPROVED** [review]

**Pull request merged**